rero_ils/modules/organisations/mappings/v6/organisations/organisation-v0.0.1.json

```diff
@@ -16,11 +16,15 @@
           "type": "keyword"
         },
         "name": {
-          "type": "text"
+          "type": "text",
+          "fielddata": true
```

Comment:
why? If you need to compute aggs or sort you can use copy_to.

Reply:
done